Makefile.am

```diff
@@ -1,27 +0,0 @@
-ACLOCAL_AMFLAGS = -I m4 ${ACLOCAL_FLAGS}
-
-AM_CFLAGS = ${WARN_CFLAGS}
-
-SUBDIRS = data gnome-initial-setup po
-
-EXTRA_DIST = \
-    autogen.sh \
```

Comment:
The reason the PR build is failing is because `autogen.sh` is still present in the repo:

```
Found autogen.sh, creating tarball with distcheck
grep: configure.ac: No such file or directory
…
+ ./autogen.sh
**Error**: Directory `.' does not look like the top-level gnome-initial-setup directory
```

As well as removing this file, please close #242 and re-open it having renamed the debian branch to `T23405-debian`. This will cause the PR builder to build the pair of branches together, so we can have reasonable confidence that they'll work when both merged.

Reply:
I arrived home (from the coworking place) and failed to see these whole changes before fixing the Debian PR name. Sorry.